Uno.Gallery/Uno.Gallery.Shared/Views/NestedPages/MaterialTopBarSampleNestedPage.xaml

```diff
@@ -42,32 +42,32 @@
 			<Grid x:Name="Home" Visibility="Collapsed">
 				<Border Background="LightGray">
```

Comment:
This Background needs to be set to `{ThemeResource SurfaceBrush}` instead of `LightGray`